src/main/scala/cromwell/engine/ExecutionStatus.scala

```diff
@@ -2,11 +2,11 @@ package cromwell.engine
 
 object ExecutionStatus extends Enumeration {
   type ExecutionStatus = Value
-  val NotStarted, Starting, Running, Failed, Done, Aborted = Value
+  val NotStarted, Starting, Running, Failed, Preempted, Done, Aborted = Value
 
   implicit class EnhancedExecutionStatus(val status: ExecutionStatus) extends AnyVal {
     def isTerminal: Boolean = {
-      Seq(Failed, Done, Aborted) contains status
+      Seq(Failed, Done, Aborted, Preempted) contains status
```

Comment:
I realize this is preexisting but we should start getting in the habit of not using `Seq()` and using `List()` instead. A call to `Seq()` *is* a call to `List()`, but it's less obvious.

Note to our resident OOP lovers (Looking at you @mcovarr and @kshakir ) - I'm not suggesting (yet) removing `Seq` from signatures